R/interface.R

```diff
@@ -467,6 +467,19 @@ get.edges <- function(graph, es) {
   ends(graph, es, names = FALSE)
 }
 
+el_to_vec <- function(x) {
```

Comment:
If there's an error, we want it to print like "Error in get_edge_ids()", not "Error in el_to_vec()" . Need a `call` argument for this, and then pass it to `cli_abort()` .

```suggestion
el_to_vec <- function(x, call = caller_env()) {
```